### PR TITLE
Add support eslint-plugin-no-unsanitized

### PR DESCRIPTION
**Pull request by mozfreddyb**

### CODE REVIEW

File: plugins.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
   "mongodb": "https://github.com/nfroidure/eslint-plugin-mongodb#RULENAME",
   "mozilla": "https://gecko.readthedocs.io/en/latest/tools/lint/linters/eslint-plugin-mozilla.html#RULENAME",
   "netguru-ember": "https://github.com/netguru/eslint-plugin-netguru-ember/blob/master/docs/RULES.md#RULENAME",
+  "no-unsanitized": "https://github.com/mozilla/eslint-plugin-no-unsanitized/blob/master/docs/rules/no-unsanitized.md",
```
Comment (j-f1) [inline plugins.json]: I think you want <code>.../docs/rules/**RULENAME**.md</code>, so it will automatically work if you add new rules.

**Review by j-f1 — CHANGES_REQUESTED** [review]

File: plugins.json (hunk 1 above)
Comment (Arcanemagus) [reply plugins.json]: Additionally it looks like you need to split your documentation up a bit, as your rules are `method` and `property`.

Comment (jonathanKingston) [reply plugins.json]: This work has now merged in: https://github.com/mozilla/eslint-plugin-no-unsanitized/pull/65

Comment (Arcanemagus) [reply plugins.json]: Great news @jonathanKingston! Now this PR just needs to be updated as @j-f1 stated so the link will be automatically filled in 😉.

Comment (mozfreddyb) [reply plugins.json]: Thanks! Done 👍

**Comment by mozfreddyb** [comment]
Should be all good now.

**Review by Arcanemagus — APPROVED** [review]
LGTM, thanks!

**Comment by Arcanemagus** [comment]
Published in v1.0.14. 🎉

**Pull request merged**